snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -3,6 +3,11 @@ use starknet::info::v2::ResourceBounds;
 use snforge_std::cheatcodes::{CheatSpan, CheatTarget, validate_cheat_target_and_span};
 use super::super::_cheatcode::handle_cheatcode;
 
+
+/// A structure used for setting individual fields in `TxInfo`
+/// All fields are optional, with optional value meaning as defined:
+/// - `None` means that the field is going to be reset to default value
```

Comment:
This can be understood as in the field will be set to default value for a type (so e.g. 0 for u32)

> `None` means that the field is not going to be spoofed and the value will be a real one

or just change `default` to `real` / `actual` or something along these lines